src/components/common/line-chart.tsx

```diff
@@ -111,12 +146,11 @@ function LineChartFactory() {
           <HorizontalGridLines tickTotal={3} />
           <LineSeries
             style={{fill: 'none'}}
```

Comment:
why is scrokeWidth removed?

Reply:
According to types from @types/react-vis package LineSeries doesn't have such property

Reply:
It is in the code, the type definition might be wrong
https://github.com/uber/react-vis/blob/master/packages/react-vis/src/plot/series/line-series.js#L80